src/_includes/head.html

```diff
@@ -1,24 +1,43 @@
-<meta prefix="og: http://ogp.me/ns#" property="og:description" content="{% if page.description %}{{ page.description }}{% else %}Our team of dedicated software craftspeople provides consultancy, software development, and training services to clients seeking high quality development processes and software solutions.{% endif %}" />
+<meta prefix="og: http://ogp.me/ns#" property="og:description" content="{% if page.url == '/software-modernisation/' and site.lang == 'es' %}{{ page.description-es }}{% elsif page.url == '/software-modernisation/' and site.lang != 'es' %}{{ page.description }}{% else %}Our team of dedicated software craftspeople provide consultancy, software development, and training services to clients seeking high quality development processes and software solutions.{% endif %}" />
 <meta prefix="og: http://ogp.me/ns#" property="og:url" content="{{ page.url | prepend: site.url }}" />
 <meta prefix="og: http://ogp.me/ns#" property="og:image" content="{% if page.image.src %}{{ page.image.src | prepend: site.url }}{% else %}{{ '/assets/custom/img/codurance_open_graph.jpg' | prepend: site.url }}{% endif %}" />
 <meta prefix="og: http://ogp.me/ns#" property="og:type" content="{% if page.title %}article{% else %}website{% endif %}" />
 <meta prefix="og: http://ogp.me/ns#" property="fb:app_id" content="1988440268098611" />
 
 <!-- Twitter Card -->
-<meta name="twitter:card" content="summary" />
-<meta name="twitter:site" content="@codurance" />
-<meta name="twitter:title" content="{% if page.title %}{{ page.title }}{% else %}Codurance{% endif %}" />
-<meta name="twitter:description" content="{% if page.description %}{{ page.description }}{% else %}Our team of dedicated software craftspeople provides consultancy, software development, and training services to clients seeking high quality development processes and software solutions.{% endif %}" />
+<meta name="twitter:card" content="summary_large_image" />
+<meta name="twitter:site" content="{% if site.lang == 'es' %}@codurance_es{% else %}@codurance{% endif %}"/>
+<meta name="twitter:title" content="{% if page.url == '/software-modernisation/' and site.lang == 'es' %}{{ page.title-es }}{% elsif page.url == '/software-modernisation/' and site.lang != 'es' %}{{ page.title }}{% else %}{% if page.title %}{{ page.title }}{% else %}Codurance{% endif %}{% endif %}" />
+<meta name="twitter:description" content="{% if page.url == '/software-modernisation/' and site.lang == 'es' %}{{ page.description-es }}{% elsif page.url == '/software-modernisation/' and site.lang != 'es' %}{{ page.description }}{% else %}Our team of dedicated software craftspeople provide consultancy, software development, and training services to clients seeking high quality development processes and software solutions.{% endif %}" />
 <meta name="twitter:image" content="{% if page.image.src %}{{ page.image.src | prepend: site.url }}{% else %}{{ '/assets/custom/img/codurance-logo-artwork.png' | prepend: site.url }}{% endif %}" />
```

Comment:
Same deal here. It feels like `page.description` should be handling all of this logic.

Reply:
Noted an as per the previous comments. Also happy to take any suggestions on how it should be implemented to support the English and Spanish versions of text plus any default values.